src/Momento.Sdk/Internal/Retry/DefaultRetryEligibilityStrategy.cs

```diff
@@ -37,35 +38,59 @@ public class DefaultRetryEligibilityStrategy : IRetryEligibilityStrategy
             //StatusCode.DataLoss,
         };
 
-        private readonly HashSet<Type> _retryableRequestTypes = new HashSet<Type>
+        private readonly HashSet<Type> _retryableRequestTypes = new()
```

Comment:
Compare to the [Go SDK version](https://github.com/momentohq/client-sdk-go/blob/9ebbbe63aefbfdfee3c0fcbe574973980838b54b/internal/retry/eligibility_strategy.go#L16).